package.json

```diff
@@ -25,6 +25,9 @@
   },
   "homepage": "https://github.com/andela/ah-frontend-sparta#readme",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.18",
```

Comment:
Why the many fontawesome packages?

Reply:
@adesege the npm install brings the entire font awesome library

Reply:
@adesege removed packages will depend on CDN instead